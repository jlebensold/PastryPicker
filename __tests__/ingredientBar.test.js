import React from 'react';
import IngredientBar from '../ingredientBar';
import renderer from 'react-test-renderer';

test('renders correctly', () => {
  const tree = renderer.create(
    <IngredientBar label={'test'} />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
