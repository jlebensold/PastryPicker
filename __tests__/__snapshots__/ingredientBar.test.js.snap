// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "flex": 1,
      "flexDirection": "column",
      "justifyContent": "flex-end",
    }
  }
>
  <View
    style={
      Object {
        "alignSelf": "flex-start",
        "flexGrow": 0,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": undefined,
        "flex": undefined,
      }
    }
  />
  <View
    style={
      Object {
        "flex": 0.2,
      }
    }
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
    >
      test
    </Text>
  </View>
</View>
`;
